### DISCUSSION TITLE
Encounter an issue when starting standalone Pulsar cluster locally

### DISCUSSION THREAD

**Discussion by Jason-xzq** [apache/pulsar#22038] [Q&A]
It raised an error saying "VM option 'UseZGC' is experimental and must be enabled via -XX:+UnlockExperimentalVMOptions." 
I'm curious which main class should I specify to resolve this issue. The guidance I followed is attached here: https://pulsar.apache.org/docs/3.1.x/getting-started-standalone/
The whole error message is attached below.
Error: VM option 'UseZGC' is experimental and must be enabled via -XX:+UnlockExperimentalVMOptions.
Error: The unlock option must precede 'UseZGC'.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

Thank you!

**Comment by lhotari** [comment]
Please upgrade your JVM version to the latest Java 17 version to run Pulsar. This is not supported on Windows. (For Windows, you would have to run Pulsar in docker.)

An alternative is to run Pulsar standalone in docker. This is useful for testing and development purposes. Please see https://pulsar.apache.org/docs/next/getting-started-docker/ for the instructions.
